### ISSUE TITLE
Plugin ignores vue.config.js webpack configuration

### ISSUE THREAD

**Issue by manuelmazzuola** [vuejs/eslint-plugin-vue#835]
Seems that this plugin ignores the webpack configuration in the `vue.config.js` file.  
If my `vue.config.js` contains:
```
module.exports = {
  configureWebpack: {
    externals: {
      config: JSON.stringify({
        backend: 'http://localhost:8081',
      }),
    },
  },
};
```

than eslint should not complain about unresolved module in that line  
`import config from 'config';`

Here my `.eslintrc.js` configuration:
```
module.exports = {
  root: true,

  env: {
    node: true,
  },

  extends: [
    'plugin:vue/recommended',
    '@vue/airbnb',
  ],

  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    quotes: [2, 'single',
      {
        avoidEscape: true,
      },
    ],
    'no-param-reassign': ['error', { 'props': true, 'ignorePropertyModificationsFor': ['state'] }]
  },

  parserOptions: {
    parser: 'babel-eslint',
  },
};
```

**Comment by mysticatea** [comment]
Thank you for this issue.

However, I'm not sure what you are saying. ESLint is not related to Webpack.

**Comment by dominik-bln** [comment]
Is the error coming from `eslint-plugin-import`? Then this might help:

https://github.com/benmosher/eslint-plugin-import/blob/a7d627ede2e4b4b7edce116b92b73703af886de5/docs/rules/named.md#settings

I don't think there is an easy way to consider the webpack config.

**Comment by ota-meshi** [comment]
I don't think it's a problem with this plugin, so I close this issue.
